sormas-backend/src/test/java/de/symeda/sormas/backend/user/UserRoleFacadeEjbTest.java

```diff
@@ -0,0 +1,31 @@
+package de.symeda.sormas.backend.user;
+
+import static org.junit.Assert.assertThrows;
+
+import org.junit.Test;
+
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Validations;
+import de.symeda.sormas.api.user.JurisdictionLevel;
+import de.symeda.sormas.api.user.UserRoleDto;
+import de.symeda.sormas.api.utils.ValidationRuntimeException;
+import de.symeda.sormas.backend.AbstractBeanTest;
+
+public class UserRoleFacadeEjbTest extends AbstractBeanTest {
+
+    @Test
+    public void testSaveUserRole() {
```

Comment:
Please rename to `testChangeUserRoleJurisdictionLevel` or something similar so it reflects what is tested by this method